lib/rules/validate-jsdoc/require-description-complete-sentence.js

```diff
@@ -0,0 +1,62 @@
+module.exports = requireDescriptionCompleteSentence;
+module.exports.scopes = ['function'];
```

Comment:
I'm not sure why we should check this only for functions. We can also check modules description, etc. Any `jsdoc` block that have `description` property.